.travis.yml

```diff
@@ -55,7 +55,7 @@ script:
   - flake8  # Enforce code style ('relaxed' line length limit is set in .flake8 config file).
   - |
     if [ ! -z "$USE_DOCKER" ]; then
-        container_id=$(docker run -d -t --rm -e SIREPO_AUTH_METHODS=bluesky:guest -e SIREPO_AUTH_BLUESKY_SECRET=bluesky -e SIREPO_SRDB_ROOT=/sirepo -e SIREPO_COOKIE_IS_SECURE=false -p 8000:8000 -v $PWD/sirepo_bluesky/tests/SIREPO_SRDB_ROOT:/SIREPO_SRDB_ROOT:ro,z radiasoft/sirepo:beta bash -c "mkdir -v -p /sirepo/user/ && cp -Rv /SIREPO_SRDB_ROOT/* /sirepo/user/ && /home/vagrant/.pyenv/shims/sirepo service http")
+        container_id=$(docker run -d -t --rm -e SIREPO_AUTH_METHODS=bluesky:guest -e SIREPO_AUTH_BLUESKY_SECRET=bluesky -e SIREPO_SRDB_ROOT=/sirepo -e SIREPO_COOKIE_IS_SECURE=false -p 8000:8000 -v $PWD/sirepo_bluesky/tests/SIREPO_SRDB_ROOT:/SIREPO_SRDB_ROOT:ro,z radiasoft/sirepo:20200220.135917 bash -l -c "mkdir -v -p /sirepo/user/ && cp -Rv /SIREPO_SRDB_ROOT/* /sirepo/user/ && sirepo service http")
```

Comment:
We may need the new command using the `beta` tag. See #17/#18.

Reply:
I had to remove `-u vagrant` but the docker command from #17 is working!